edgefirewall.yaml

```diff
@@ -549,90 +549,98 @@ components:
           type: boolean
 
     Behaviors:
+      oneOf:
+        - $ref: '#/components/schemas/NullArgumentBehavior'
+        - $ref: '#/components/schemas/SimpleArgumentBehavior'
+        - $ref: '#/components/schemas/SetRateLimitBehavior'
+        - $ref: '#/components/schemas/SetWAFRuleSetBehavior'
+
+    NullArgumentBehavior:
       type: object
       properties:
         name:
           type: string
-          enum: [deny, drop, set_rate_limit, set_waf_ruleset, run_function, tag_event]
+          enum: [deny, drop]
```

Comment:
There is a situation where a null `argument` comes in a `deny` or `drop` behavior? In my tests, only the `name` field came in these behaviors.

Reply:
<img width="616" alt="Captura de Tela 2023-10-13 às 16 59 20" src="https://github.com/aziontech/azionapi-openapi/assets/8815139/c989a42d-87a6-4bae-9b49-21b43d730273">

Reply:
<img width="901" alt="Captura de Tela 2023-10-13 às 16 59 57" src="https://github.com/aziontech/azionapi-openapi/assets/8815139/9c72f346-0c5b-441a-b229-8532ae760e9d">

Reply:
Yep, probably. The tests I made were via UI, and, in these cases, the `argument` field was not present 😞

Reply:
Taking a look at open api doc, unless we explicitly mark a field as required it will be optional